openid-4-verifiable-presentations-1_0.md

```diff
@@ -460,6 +484,64 @@ To use `client_id_scheme` values `entity_id`, `did`, `verifier_attestation`, `x5
 
 Other specifications can define further values for the `client_id_scheme` parameter. It is RECOMMENDED to use collision-resistant names for such values.
 
+## Request URI Method POST {#request_uri_method_post}
+
+This request is handled by the Request URI endpoint of the Verifier.  
+
+The request MUST use the HTTP POST method with the https scheme, and the content type "application/x-www-form-urlencoded" and the accept header set to "application/oauth-authz-req+jwt". 
```

Comment:
```suggestion
The request MUST use the HTTP POST method with the https scheme, and the content type "application/json" and the accept header set to "application/oauth-authz-req+jwt". 
```

Reply:
the content type is correct, we follow the OAuth philosophy of form encoded requests and JSON payload in the response (see direct post)

Reply:
Maybe this document belongs in the OAUTH WG, if we are going to follow OAUTH philosophy.

I'm a -1 to form url encoding in new APIs, but it the goal is consistency with OAUTH I could live with that assuming review from OAUTH experts.

Happy to be in the rough on this though.

Reply:
as I said, the direct post endpoint is form encoded, too. If we change this to JSON, we must change the direct post as well.

Reply:
i don't think we should prevent an option for the wallet to be able to send its attestation to the verifier using client auth in which case form url encoded is probbaly more future proofing

Reply:
WG call: suggest to leave as is and discuss potential move to JSON encoding in an issue.

Reply:
filed issue #132